🐞 [Bug]: nft-enumerable does not build a valid javascript package

### What happened?

Steps:
- cd into `nft-enumerable`
- compile the example `cargo build --target wasm32v1-none --release`
- stellar contract bindings typescript --wasm target/wasm32v1-none/release/nft_enumerable_example.wasm --output-dir <output dir>
- cd <output dir>
- npm i && npm run build

Expected: Should be able to build the generated package into a client library
Actual: Get build errors because `DataKey` is used multiple times:

```
$ npm run build

> <output-dir>@0.0.0 build
> tsc

src/index.ts:36:13 - error TS2300: Duplicate identifier 'DataKey'.

36 export type DataKey = {tag: "Owner", values: void};
               ~~~~~~~

src/index.ts:504:13 - error TS2300: Duplicate identifier 'DataKey'.

504 export type DataKey = {tag: "HookModules", values: readonly [ComplianceHook]};
                ~~~~~~~


Found 2 errors in the same file, starting at: src/index.ts:36
```

It seems the generated binding has various types from the crates included as dependencies; this DataKey is from the RWA storage example.

I'm not sure if the fix lies with Stellar CLI bindings, I'm opening the issue here in case it might be fixed in this context because of changes to the crates that are included.

### Expected behavior

_No response_

--- a/packages/tokens/src/rwa/compliance/storage.rs
+++ b/packages/tokens/src/rwa/compliance/storage.rs
@@ -11,7 +11,7 @@ use crate::rwa::{
 /// Storage keys for the modular compliance contract.
 #[contracttype]
 #[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
-pub enum DataKey {
+pub enum ComplianceDataKey {
     /// Maps ComplianceHook -> `Vec<Address>` for registered modules
     HookModules(ComplianceHook),
 }
@@ -33,7 +33,7 @@ pub enum DataKey {
 /// A vector of module addresses registered for the specified hook.
 /// Returns an empty vector if no modules are registered.
 pub fn get_modules_for_hook(e: &Env, hook: ComplianceHook) -> Vec<Address> {
-    let key = DataKey::HookModules(hook);
+    let key = ComplianceDataKey::HookModules(hook);
     if let Some(existing_modules) = e.storage().persistent().get(&key) {
         e.storage().persistent().extend_ttl(
             &key,
@@ -110,7 +110,7 @@ pub fn add_module_to(e: &Env, hook: ComplianceHook, module: Address) {
     }
 
     // Add the module
-    let key = DataKey::HookModules(hook.clone());
+    let key = ComplianceDataKey::HookModules(hook.clone());
     modules.push_back(module.clone());
     e.storage().persistent().set(&key, &modules);
 
@@ -159,7 +159,7 @@ pub fn remove_module_from(e: &Env, hook: ComplianceHook, module: Address) {
     modules.remove(index);
 
     // Update storage
-    let key = DataKey::HookModules(hook.clone());
+    let key = ComplianceDataKey::HookModules(hook.clone());
     e.storage().persistent().set(&key, &modules);
 
     // Emit event

